tsl/src/chunk.c

```diff
@@ -421,3 +422,134 @@ chunk_drop_replica(PG_FUNCTION_ARGS)
 
 	PG_RETURN_VOID();
 }
+
+typedef struct
+{
+	Chunk *chunk;
+	NameData operation_id;
+	List *src_node;
+	List *dst_node;
+} ChunkCopyData;
```

Comment:
I believe several comments to the structure and its fields would be appropriate.